app/assets/src/components/views/discovery/table_renderers.scss

```diff
@@ -182,7 +190,7 @@
     }
   }
 
-  .publicAccess {
+  .visibility {
```

Comment:
not sure exactly what line changed this, but it looks like the icons don't have enough padding now? 

![Screen Shot 2019-09-19 at 5 30 01 PM](https://user-images.githubusercontent.com/5652739/65290243-550fe280-db03-11e9-8e90-8aa245916848.png)